src/main/java/org/apache/commons/compress/archivers/zip/ZipSplitOutputStream.java

```diff
@@ -65,7 +65,7 @@ public ZipSplitOutputStream(final File zipFile, final long splitSize) throws Ill
         this.zipFile = zipFile;
         this.splitSize = splitSize;
 
-        this.outputStream = new FileOutputStream(zipFile);
+        this.outputStream = Files.newOutputStream((zipFile.toPath()));
```

Comment:
please remove the redundant pair of parenthesis

Reply:
Changed

Reply:
thank you